src/main/java/seedu/fridgefriend/command/ListCommand.java

```diff
@@ -74,27 +106,54 @@ public String getListAllMessage() {
     private String getFoodDescription(int index) {
         int indexShownToUser = index + EXTRA_INDEX;
         Food food = fridge.getFood(index);
-        String foodDescription = 
-                "\n\t" + indexShownToUser +  ". "
-                + food.getFoodName() + " ["
-                + food.getCategory() + "]";
+        String foodDescription =
+                "\n\t" + indexShownToUser + ". "
+                        + food.getFoodName() + " ["
+                        + food.getCategory() + "]";
         return foodDescription;
     }
 
-    private void checkIsValidCategory() throws InvalidInputException {
-        if (!FoodCategory.contains(category)) {
-            String errorMessage = "Sorry my friend, please enter a valid food category.";
-            throw new InvalidInputException(errorMessage);
+    private boolean checkIsValidCategory() {
```

Comment:
I think you can just return FoodCategory.contains(description) here

Reply:
Hmm I think not necessary? better to abstract/SLAP out leh

Reply:
Good catch, resolved